modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -1931,19 +1944,19 @@ private HybridTimestampMessage hybridTimestamp(HybridTimestamp tmstmp) {
      *
      * @param tablePartId {@link TablePartitionId} object to construct {@link UpdateCommand} object with.
      * @param rowUuid Row UUID.
-     * @param rowBuf {@link ByteBuffer} representation of {@link BinaryRow}.
+     * @param row {@link BinaryRow}.
      * @param txId Transaction ID.
      * @return Constructed {@link UpdateCommand} object.
      */
-    private UpdateCommand updateCommand(TablePartitionId tablePartId, UUID rowUuid, ByteBuffer rowBuf, UUID txId) {
+    private UpdateCommand updateCommand(TablePartitionId tablePartId, UUID rowUuid, TableRow row, UUID txId) {
```

Comment:
why did you change method's signature?

Reply:
Mainly to reduce clutter at the call site, but probably it would be better if the update and updateAll singatures would be similar.